Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1309,6 +1326,32 @@ def get_workflow_usage(owner_name, repository_name, workflow_id):
     return workflow_usage
 
 
+def list_team_members_command():
+    args = demisto.args()
+    org = args.get('organization')
+    team_slug = args.get('team_slug')
+    maximum_users = int(args.get('maximum_users'))
+    response = get_team_members(org, team_slug, maximum_users)
+    members = []
+    for member in response:
+        context_data = {
+            'ID': member.get("id"),
+            'Login': member.get("login")
```

Comment:
better to add the team name to the context object as well and [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
            'Login': member.get("login"),
            'Team': team_slug,
```